src/main.qml

```diff
@@ -17,45 +18,46 @@
 
 import QtQuick 2.9
 import org.asteroid.controls 1.0
+import org.asteroid.utils 1.0
 import Nemo.KeepAlive 1.1
 
 Application {
-    centerColor: "#b04d1c"
-    outerColor: "#421c0a"
+    id: app
+
+    centerColor: "#00A698"
+    outerColor: "#000C07"
 
     Rectangle {
-        id: whiteOverlay
-        color: "#fff"
-        anchors.fill: parent
-        OpacityAnimator {
-            id: onOffAnimation
-            target: whiteOverlay
-            to: 1
-            duration: 100
-        }
-    }
+        id: flashCircle
 
-    Icon {
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.verticalCenter: parent.verticalCenter
-        width: Dims.l(20)
+        property bool flashOn: true
+
+        anchors.centerIn: app
+        color: flashOn ? "#ffffffff" : "#66444444"
+        width: flashOn ? Dims.w(100) : Dims.w(45)
         height: width
-        name:  "ios-bulb"
-    }
+        radius: DeviceInfo.hasRoundScreen ? width : flashOn ? 0 : width
 
-    MouseArea {
-        anchors.fill: parent
-        onClicked: {
-            if (onOffAnimation.to == 1) {
-                onOffAnimation.from = 1
-                onOffAnimation.to = 0
-                onOffAnimation.start()
-            } else {
-                onOffAnimation.from = 0
-                onOffAnimation.to = 1
-                onOffAnimation.start()
+        Icon {
+            anchors {
+                centerIn: flashCircle
+                verticalCenterOffset: Dims.h(1)
```

Comment:
Dims.h is a percentage of the screen size so the offset will be the same whether the icon is large (flashOn) or not. This feels a bit wrong
If this is just 1% of the overall screen and you didn't notice the discrepancy, I'm tempted to say this is not necessary... :p 
And if the icon is not well aligned I think it's best to fix the icon than the app with this kind of hard-coded offsets.

Reply:
Well, it is intentional to have a proportionally larger offset when the icon is small and proportionally small offset when the icon is large :P
But now that you are uncomfortable with how this is implemented, i'll pick the clean option to center the icons for the usage in narrow spaced circles (!flashOn) and ignore the change for flashOn.
Original intention was to offset the icon 1% when small, and "don't care" about the offset when enlarged/flashOn.
Which is also reached by fixing the icon scg.
Reverting the offset here so we can merge.
Fixing the icons in another PR.

Reply:
> Well, it is intentional to have a proportionally larger offset

Yes, but notice that Dims.h is proportional to... the screen size. Which is constant. So 

> when the icon is small and proportionally small offset when the icon is large :P

So this is not happening in practice. The offset is always the same: 1% of your screen height